Added Totivirus celatus

Dangelo2002 opened the pull request:
I know it's kinda late. Not for bonus, just for fun :). The video is not included.

Commit:
added Totivirus celatus

Commit:
updated image links and discussions

ababaian commented on virus/img/Totivirus_celatus/image_summary.png:
This image is a bit large, can you make it lower res for the repo. Ideally ~1MB.

ababaian reviewed (changes requested):
Great PR! I'll give you partial marks for doing it if you can close the PR. Even just for fun :)

Commit:
Updated image and GIF->MP4 to reduce size

Dangelo2002 committed:
Update MP4 rdrp visualization

Dangelo2002 committed:
Fixed MP4 visualization

Dangelo2002 committed:
Merge branch 'main' into main

Dangelo2002 commented:
Done. I have compressed this image into a ~0.7 MB JPG file and replaced the embedding codes. I also replaced the 10 MB GIF file with a smaller integrated MP4 player link. By the way, there are two unused images (alphafold.png and serratus.png) in the folder, which were only used in "Tweet-torial" for my final project submission to GitLab. Please let me know if other changes are needed.
